### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by tronical** [corrosion-rs/corrosion#273] [General]
Hi :)

I was wondering if there are any plans for a new release soon. We're going to use a git version for the `CMAKE_*_OUTPUT_DIRECTORY` support, but would love to use a released version if available.

**Comment by jschwe** [comment]
Sure, I can prepare a bugfix release.

**Reply by jschwe** [reply]
Update: Released v0.3.1 and updated the stable/v0.3 branch.

**Reply by tronical** [reply]
Oh that was quick! Thank you so much!
